Dhruv-Mathur/Dhruv-Mathur.md

```diff
@@ -33,7 +33,11 @@ quoteAuthor: "Jim Rohn"
 [[imgBadge]]
 | ![.NET Core](../badges/Developer-dotnet-core.png)
 
-Dhruv Mathur is a skilled software developer at SSW, specializing in creating outstanding software solutions using modern technologies such as .NET Core, ReactJS, Angular, Blazor, NextJS, ASP .NET Identity and IdentityServer. He enjoys working in a learning-focused environment at SSW, where he has the opportunity to enhance both his technical expertise and professional communication skills.
+Dhruv Mathur is our go-to software developer here at SSW, especially when it comes to anything .NET MVC, .NET Web APIs, ASP.NET, and .NET Core. He's got a real talent for piecing together applications that are not just powerful, but also user-friendly. Thanks to his solid skills with Angular, React, and the essentials like CSS, Bootstrap, Tailwind, and HTML5, the apps he works on are easy on the eyes and a breeze to use.
```

Comment:
```suggestion
Dhruv is a full stack dev who specializes in .NET MVC, .NET Web APIs, ASP.NET, and .NET Core. He focuses on piecing together applications that are not just powerful, but also user-friendly. Thanks to his solid skills with Angular, React, and the essentials like CSS, Bootstrap, Tailwind, and HTML5, the apps he works on are easy on the eyes and a breeze to use.
```

1. Removed - SSW doesn't have a go to developer at all 😂
2. Making it sound less casual